### PR TITLE
Added Inception Module to the Model.

### PR DESCRIPTION
**Pull request by ujjwal-9**
The module basically acts as multiple convolution filter inputs, that are processed on the same input. It also does pooling at the same time. All the results are then concatenated. This allows the model to take advantage of multi-level feature extraction from each input. For instance, it extracts general (5x5) and local (1x1) features at the same time.

### CODE REVIEW

File: models/cnn.py [hunk 1]
```diff
@@ -61,6 +84,11 @@ def create_network(img_height, img_width, num_classes):
     # layer = tf.nn.dropout(layer, 0.75)
     layer = tf.nn.relu(layer)
 
+    # Inception Module
+    layer = inception2d(input=layer,
+                         num_channels=16,
```
Comment (amaneureka) [inline models/cnn.py]: Don't you think num_channels should be 10 and num_filter=16

**Comment by amaneureka** [comment]
We are running code on [FloydHub](https://www.floydhub.com)


    floyd run "python train.py --dataset /input --savedir /output --train" --data WLVWALmiVEmDJRyd5c6YgY --gpu

**Comment by ujjwal-9** [comment]
I have corrected the issue. :)

**Comment by amaneureka** [comment]
what version of tensorflow you are using ?

```
2017-05-09 23:54:37,889 INFO - Traceback (most recent call last):
2017-05-09 23:54:37,890 INFO - File "train.py", line 77, in <module>
2017-05-09 23:54:37,891 INFO - x, y, y_true, optimizer = cnn.create_network(height, width, num_classes)
2017-05-09 23:54:37,891 INFO - File "/code/models/cnn.py", line 90, in create_network
2017-05-09 23:54:37,892 INFO - num_filter=16)
2017-05-09 23:54:37,892 INFO - File "/code/models/cnn.py", line 51, in inception2d
2017-05-09 23:54:37,893 INFO - input = tf.concat([one_by_one, three_by_three, five_by_five, pooling], axis=3)  # Concat in the 4th dim to stack
2017-05-09 23:54:37,894 INFO - TypeError: concat() got an unexpected keyword argument 'axis'
```

**Comment by amaneureka** [comment]
This worked
```
diff --git a/floyd_requirements.txt b/floyd_requirements.txt
new file mode 100644
index 0000000..79ced33
--- /dev/null
+++ b/floyd_requirements.txt
@@ -0,0 +1 @@
+tensorflow==1.0.1
```

**Comment by ujjwal-9** [comment]
:)

**Pull request merged**